### PR TITLE
Config update

### PR DESCRIPTION
**Pull request by tok-kkk**
- Update config format to the latest version in darknode.
- Remove `--config` flag form the `update` command to avoid conflicts in the config file. 
- Show messages to tell if nodes have successfully been started/stopped/restarted.
- Minor tweak with the darknode installation script.
- Fix incorrect bootstrap addresses on testnet and devnet.

### CODE REVIEW

File: cmd/switch.go [hunk 1]
```diff
@@ -41,6 +42,11 @@ func updateServiceStatus(ctx *cli.Context, cmd string) error {
 	errs := make([]error, len(nodes))
 	phi.ParForAll(nodes, func(i int) {
 		errs[i] = util.RemoteRun(nodes[i], script)
+		if errs[i] == nil {
+			color.Green("[%v] has been %v.", nodes[i], message)
+		} else {
+			color.Red("fail to %v [%v], err = %v", script, nodes[i])
```
Comment (jazg) [inline cmd/switch.go]: "failed to %v [%v]: %v"

Also I think you might have forgotten to actually include the error (`errs[i]`) in the format string.

File: cmd/down.go [hunk 2]
```diff
@@ -26,53 +25,55 @@ import (
...
+		switch st {
+		case pendingRegistration, pendingDeregistration, registered, notRefunded:
+			color.Red(st.err())
+			color.Red("Please try again after fully node has been fully deregistered and refunded.")
+			return nil
+		default:
+		}
 
+		// Last time confirm with user.
+		fmt.Println("Do you really want to destroy your darknode? (Yes/No)")
```
[the remark this answers is not in the document]
Comment (tok-kkk) [reply cmd/down.go]: It accepts `y` and `yes` (case insensitive) and returns for all others.

File: cmd/switch.go (hunk 1 above)
Comment (tok-kkk) [reply cmd/switch.go]: Yes, you are right. Fixed.

File: cmd/down.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (jazg) [reply cmd/down.go]: Sweet. Can you change this to "Are you sure you want to destroy your Darknode? (y/N)"

**Pull request merged**